employees/views.py

```diff
@@ -514,12 +523,55 @@ def render_to_response(self, context: dict, **response_kwargs: Any) -> HttpRespo
                 zip_file.getvalue(), content_type=ExcelGeneratorSettingsConstants.ZIP_CONTENT_TYPE_FORMAT.value
             )
             response["Content-Disposition"] = ExcelGeneratorSettingsConstants.ZIP_EXPORTED_FILE_NAME.value.format(
-                project.name, datetime.date.today()
+                project.name, f"{self.kwargs['month']}/{self.kwargs['year']}"
+            )
+        else:
+            response = HttpResponse(content_type=ExcelGeneratorSettingsConstants.XLSX_CONTENT_TYPE_FORMAT.value)
+            response["Content-Disposition"] = ExcelGeneratorSettingsConstants.XLSX_EXPORTED_FILE_NAME.value.format(
+                project.name, f"{self.kwargs['month']}/{self.kwargs['year']}"
+            )
+            work_book.save(response)
+        return response
+
+
+@method_decorator(login_required, name="dispatch")
+@method_decorator(
+    check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name, CustomUser.UserType.MANAGER.name]),
+    name="dispatch",
+)
+class ExportAuthorReportProjectView(UserIsManagerOfCurrentProjectMixin, DetailView):
+    model = Project
+
+    def get_queryset(self) -> QuerySet:
+        return (
+            super()
+            .get_queryset()
+            .prefetch_related(
```

Comment:
Use for it the same method defined here https://github.com/Code-Poets/sheetstorm/pull/424/files#r302780834 but add extra param to it for author.